src/js/components/search/topFilterBar/filterGroups/PSCFilterGroup.jsx

```diff
@@ -36,23 +36,23 @@ export default class PSCFilterGroup extends React.Component {
     }
 
     generateTags() {
-        const tags = [];
-
         // check to see if a PSC code is provided
-        const PSC = this.props.filter.values;
-
-        PSC.forEach((value) => {
-            const tag = {
+        return this.props.filter.values.map((value) => {
+            if (value.isV2) {
```

Comment:
Yes... I created a ticket to remove dead code for next sprint in case for some reason we need to revert this release.

https://federal-spending-transparency.atlassian.net/browse/DEV-5161